S32-num/rounders.t

```diff
@@ -75,6 +75,27 @@ for @testkeys -> $type {
     }
 }
 
+# Tests for 2-argument version of &round
+# (https://github.com/rakudo/rakudo/issues/1745).
+{
+  my @tests = (
+    [ 1.23456, 0.00001, 1.23456 ],
+    [ 1.23456, 0.0001,  1.2346  ],
+    [ 1.23456, 0.001,   1.235   ],
+    [ 1.23456, 0.01,    1.23    ],
+    [ 1.23456, 0.1,     1.2     ],
+
+    # Make sure first argument is coerced to Numeric.
+    [ "123.12", 0.01, 123.12 ],
+    [ [<foo bar baz>], 1, 3 ],
+    [ { :1foo, :2bar, :3baz }, 1, 3 ],
+  );
+
+  for @tests {
+    ok(round(.[0], .[1]) == .[2], "round({.[0].perl}, {.[1].perl}) == {.[2].perl}");
```

Comment:
It's fine here, since you included `.perl`s in the test description, but if you use a more appropriate test routine, you won't have to do it, as the routine will print out the values for you:

```perl6
$ perl6 -MTest -e 'cmp-ok round("42"), "==", 100, "rounding a Str"'
not ok 1 - rounding a Str
# Failed test 'rounding a Str'
# at -e line 1
# expected: 100
#  matcher: 'infix:<==>'
#      got: 42
```